pages/checkLoginSucceeded.py

```diff
@@ -0,0 +1,15 @@
+
+class Actual:
```

Comment:
Why is the class name still actual?

Reply:
my bad, i changed the filename but forgot to change the class name